src/main/scala/io/delta/tables/execution/VacuumTableCommand.scala

```diff
@@ -44,8 +44,7 @@ case class VacuumTableCommand(
     val pathToVacuum =
       new Path(if (table.nonEmpty) {
         DeltaTableIdentifier(sparkSession, table.get) match {
-          case Some(id) if id.path.isDefined => id.path.get
-          case Some(id) => throw DeltaErrors.tableNotSupportedException("VACUUM")
+          case Some(id) => id.path.getOrElse{throw DeltaErrors.tableNotSupportedException("VACUUM")}
```

Comment:
same here. there should be spaces. before and after `{` and `}`.

Reply:
done 👍